package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.2",
+  "version": "1.1.2",
```

Comment:
please don't bump versions in pull-requests, allow for the package maintainer to do so as there might be more fixes / merges that would occur at the time of merging you changes :)